website-components-playground/src/pages/tabular-modal-page.js

```diff
@@ -50,105 +41,126 @@ const exampleCustomTitleRow = (
+              initialValue: 'default',
+            },
+            {
+              kind: 'select',
+              name: 'hideControls',
+              label: 'Hide Controls?',
+              valueOptions: [
+                { value: false, label: 'No' },
+                { value: true, label: 'Yes' },
+              ],
+              initialValue: true,
+            },
+          ]}
         >
-          {({ isOpen, setIsOpen }) => (
-            <TabularModalPage
+          {({ values }) => (
+            <ModalController
               title={values.title}
-              isOpen={isOpen}
-              onClose={() => setIsOpen(false)}
-              getParentSelector={() =>
-                document.querySelector(`#${containerId}`)
-              }
-              // FIXME: use proper Tab components
-              tabControls={
-                <TabControlsContainer>
-                  <span>
-                    <Text.Subheadline as="h4">Tab One</Text.Subheadline>
-                  </span>
-                  <span>
-                    <Text.Subheadline as="h4">Tab Two</Text.Subheadline>
-                  </span>
-                  <span>
-                    <Text.Subheadline as="h4">Tab Three</Text.Subheadline>
-                  </span>
-                </TabControlsContainer>
-              }
-              customTitleRow={
-                values.useCustomTitleRow === 'custom' && exampleCustomTitleRow
-              }
-              formControls={
-                <>
-                  <TabularModalPage.FormSecondaryButton
-                    onClick={() => {
-                      setIsOpen(false);
-                    }}
-                  />
-                  <TabularModalPage.FormPrimaryButton
-                    onClick={() => undefined}
-                    isDisabled
-                  />
-                  <TabularModalPage.FormDeleteButton
-                    onClick={() => undefined}
-                    isDisabled
-                  />
-                </>
-              }
-              hideControls={values.hideControls}
+              buttonLabel="Open Tabular Modal Page"
+              containerId={containerId}
             >
-              <Text.Body>{values.content}</Text.Body>
-            </TabularModalPage>
+              {({ isOpen, setIsOpen }) => (
+                <TabularModalPage
+                  title={values.title}
+                  isOpen={isOpen}
+                  onClose={() => setIsOpen(false)}
+                  getParentSelector={() =>
+                    document.querySelector(`#${containerId}`)
+                  }
+                  tabControls={
+                    <>
+                      <TabHeader to="/tab-one" label="Tab One" />
+                      <TabHeader to="/tab-two" label="Tab Two" />
+                      <TabHeader
+                        to="/tab-three"
+                        label="Disabled tab"
+                        isDisabled
+                      />
```

Comment:
I think these should be updated as well

```suggestion
                      <TabHeader to={withPrefix('/tabular-modal-page/tab-one')} label="Tab One" />
                      <TabHeader to={withPrefix('/tabular-modal-page/tab-two')} label="Tab Two" />
                      <TabHeader
                        to={withPrefix('/tabular-modal-page/tab-three')}
                        label="Disabled tab"
                        isDisabled
                      />
```